Sources/App/Models/Social/Social+Net.swift

```diff
@@ -0,0 +1,66 @@
+import Foundation
+
+extension Social {
+
+  struct Nets {
+    static let fb = "facebook"
+    static let vk = "vkontakte"
+    static let git = "github"
```

Comment:
git and GitHub are different things, so i suggest to rename `git` to `github`

Reply:
agree, fixed